knowledge-portal/source/README.md

```diff
@@ -1,61 +1,67 @@
-# Usage
+# Knowlege portal Source Pipeline
 
-## Adding this pipeline to your repo
+## Usage
+
+### Adding this pipeline to your repo
 
 You will have to modify the existing `gitlab-ci.yml` file to include:
 
 ```yaml
-    stages:
-        - push
-    include:
-        - remote: 'https://github.com/detecttechnologies/Gitlab-CI-CD-Templates/raw/main/knowledge-portal/source/.gitlab-ci.yml'
+stages:
+  - push 
+
+include:
+  - remote: 'https://github.com/detecttechnologies/Gitlab-CI-CD-Templates/raw/main/knowledge-portal/source/.gitlab-ci.yml'
+
+variables:
+  GIT_STRATEGY: clone
 
 ```
-## Configure CI/CD variables (applied on overall gitlab instance)
+### Using the toml file to connect to Knowledge Portal
 
-1. Go to `Settings >> CI/CD >> Variables`
-2. Click on `Add variable`. 
-3. Give the name `CENTRAL_GIT_PUSH_URL` in `Key` field and `Value` field will be provided by your maintainers.
-    * For maintainers: The `CENTRAL_GIT_PUSH_URL` will be of the form: `https://oauth2:$PUSH_TOKEN@gitlab.com/<org-name>/<path-to-central-git-repo>.git`
-4. Click on `Add variable` button to confirm changes. 
+Here is an example of a `docs-manifest.toml` file that you can use to create your own and connect your pipeline to knowledge portal:
 
 
-> The steps of this section are being performed on the overall gitlab instance, so that it need not be replicated for every repo.
+```toml
+# "includes" section maps source paths to paths at Knowledge Portal
```

Comment:
Can perhaps reword to make it easier to read. E.g.

"includes" section maps paths inside your source Git repository to paths that the documents should take within the Knowledge Portal

Reply:
done